Polygon_mesh_processing/test/Polygon_mesh_processing/orient_polygon_soup_test.cpp

```diff
@@ -4,7 +4,9 @@
 #include <CGAL/Polyhedron_3.h>
 #include <CGAL/Surface_mesh.h>
 
+#include <CGAL/Polygon_mesh_processing/orientation.h>
 #include <CGAL/Polygon_mesh_processing/orient_polygon_soup.h>
+#include <CGAL/Polygon_mesh_processing/orient_polygon_soup_extension.h>
```

Comment:
@maxGimeno @sloriot Why are there three documented headers, instead of one?

Reply:
Avoid bringing AABB-tree as a dependency (it is only a dependency of `_extension`)